Makefile

```diff
@@ -45,7 +45,7 @@ test: ## Runs unit test
 	go test -covermode atomic  `go list ./... | grep -v e2e`
 
 golangci-lint:
-	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.39.0
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.40.0
```

Comment:
Bumping the version because 1.40.0 contains a fix for when the wrapcheck linter was flagging `errors.Wrap()` as needing wrapping (see [issue](https://github.com/tomarrell/wrapcheck/issues/12)).